abel/tools/symmetry.py

```diff
@@ -31,6 +35,16 @@ def get_image_quadrants(IM, reorient=True, symmetry_axis=None,
        Include quadrant (Q0, Q1, Q2, Q3) in the symmetry combination(s)
        and final image
 
+    symmetrize: str
+       Method used for symmetrizing the image.      
+ 
+       average: Simply average the quadrants.
+       fourier: Axial symmetry implies that the Fourier components of the 2-D projection 
+                should be real and even. Removing the imaginary components in reciprocal 
```

Comment:
Should this be reworded? I think that axial symmetry implies that we have an "even" function, which implies that the Fourier components are real, but makes no even/odd statement about the Fourier components. Probably you can just remove the word "even" here. Maybe I misunderstand...